ingestions/ingest_csvs.py

```diff
@@ -187,20 +304,68 @@ def main(argv=None):
     :param csvfile: CSV file with ingestion information
     :param ingest_type: specifies either a telemetered (recurring) or recovered
         (once only) ingest request type.
+    :param netrc_account: string of the user's netrnc "machine" account
+        to use for authentication.  Either the full machine name or the
+        username when the .netrc machine name is formatted
+        as username@domain
     :param begin_date: the data must be generated by the instrument after this
         date and time (format is 'yyyy-mm-dd' or 'yyyy-mm-dd HH:MM:SS')
     :param end_date: the data must be generated by the instrument before this
         date and time (format is 'yyyy-mm-dd' or 'yyyy-mm-dd HH:MM:SS')
     :return: None, though results are saved as a CSV file in the directory the
         command is called from.
+
+    Notes
```

Comment:
Remove two of the three instances of this text. Doesn't need to be repeated 3 times. Suggest removing from here and the epilog. Suggest leaving it in the epilog as that will print if the user uses the --help option.